app/code/Magento/InventoryApi/Test/Api/StockRepository/ValidationTest.php

```diff
@@ -100,10 +102,11 @@ public function testFailedValidationOnCreate(string $field, $value, array $expec
     }
 
     /**
-     * @param string $field
+     * @param string      $field
```

Comment:
Remove extra spaces
we use only one space between type and variable name

Reply:
done.